frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -209,6 +209,11 @@ export default function ClubEditCard({
         </Text>
       ),
       fields: [
+        {
+          name: 'address',
+          required: false,
+          type: 'address',
```

Comment:
Could you change this from "Address" to "Location"? Since it is not always an address, location is a more apt descriptor.

Also, could you add the follow help text below:
```
Remember, this will be available to the public. Please only include information you feel comfortable sharing.
```

Also change the label instance below.